Xamarin.Essentials/Geolocation/Geolocation.win32.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Device.Location;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Geolocation
```

Comment:
Consider using the com interfaces for this, we avoid having a dependency for this. The actual implementation uses the COM service as well. Very Optional though as the C# wrapper is quite a lot easier to deal with.

Reply:
@Mrnikbobjeff The System.Devices dll is a built in part of the .NET framework so doesn't introduce a dependency, using the COM interfaces will require more code to achieve the same result.